### PR TITLE
fix: look for external services in CfApps file instead of the CfServices file

### PR DESCRIPTION
**Pull request by HarikrishnanBalagopal**
Also change the python code to use the new input and output format.

Signed-off-by: Harikrishnan Balagopal <harikrishmenon@gmail.com>

### CODE REVIEW

File: assets/built-in/transformers/kubernetes/operatorsfromtca/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # FROM quay.io/konveyor/tca-service-api
 FROM quay.io/mihirc36/tackle-container-advisor
 
-COPY detect.py transform.py ./
+COPY detect.py transform.py catalogue.yaml ./
```
Comment (ashokponkumar) [inline assets/built-in/transformers/kubernetes/operatorsfromtca/Dockerfile:4]: How can users configure this catalogue?

**Review by ashokponkumar — CHANGES_REQUESTED** [review]
Couple of questions.

File: assets/built-in/transformers/kubernetes/operatorsfromtca/Dockerfile (hunk 1 above)
Comment (HarikrishnanBalagopal) [reply assets/built-in/transformers/kubernetes/operatorsfromtca/Dockerfile:4]: By using a custom transformer. They can replace it with their own catalogue.yaml

Comment (ashokponkumar) [reply assets/built-in/transformers/kubernetes/operatorsfromtca/Dockerfile:4]: users should be able to specify a different catalogue for each run, and that should not require changing the custom transformer image.

Comment (HarikrishnanBalagopal) [reply assets/built-in/transformers/kubernetes/operatorsfromtca/Dockerfile]: done

**Review by ashokponkumar — APPROVED** [review]

**Pull request merged**